seatunnel-common/src/main/java/org/apache/seatunnel/common/exception/CommonErrorCode.java

```diff
@@ -62,7 +62,15 @@ public enum CommonErrorCode implements SeaTunnelErrorCode {
             "COMMON-28",
             "'<identifier>' array type not support genericType '<genericType>' of '<fieldName>'"),
     UNSUPPORTED_ROW_KIND(
-            "COMMON-29", "'<identifier>' table '<tableId>' not support rowKind  '<rowKind>'");
+            "COMMON-29", "'<identifier>' table '<tableId>' not support rowKind  '<rowKind>'"),
+
+    WRITE_SEATUNNEL_ROW_ERROR_WITH_SCHEMA_INCOMPATIBLE_SCHEMA(
+            "COMMON-30",
+            "<connector>: The source filed with sql schema '<sourceFieldSqlSchema>', except sql schema of sink is '<exceptFieldSqlSchema>'; but the filed in sink table which actual sql schema is '<sinkFieldSqlSchema>'. Please check schema of sink table."),
```

Comment:
Not all connectors use sql.
```suggestion
            "<connector>: The source filed with schema '<sourceFieldSqlSchema>', except schema of sink is '<exceptFieldSqlSchema>'; but the filed in sink table which actual schema is '<sinkFieldSqlSchema>'. Please check schema of sink table."),
```

Reply:
Done